docs/AnalyseTechnologique.md

```diff
@@ -0,0 +1,34 @@
+This document describes a comparison of technologies to develop a Desktop application on Windows which will lead to a choice to develop the project. Only technologies based on dotNet will be considered here.
+
+# Frameworks for the UI
+Here it will be discussed the choice of the framework used for the graphical interface
+
+## Deprecated Frameworks
+
+### WinForms
+WinForms (or Windows Forms) has long been the default technology for Windows dotNET development. However, the project has been in maintenance mode since 2014, so no features have been added since. It was therefore decided to use more modern technologies.
```

Comment:
Could you add some links to the corresponding docs (like it was done with Fluent design) ?

Reply:
Done